src/user/ZulipPasswordAuthView.js

```diff
@@ -20,7 +20,7 @@ import {
   attemptLogin,
 } from './userActions';
 
-export class ZulipPasswordAuthView extends React.Component {
+export class PasswordAuthView extends React.Component {
```

Comment:
Ah thanks. I don't think we need to export this component, just the version wrapped with connect.

For the future, I agree that we don't need the Zulip prefix and can remove it from all of the components.